libs/sdk-macros/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "sdk-macros"
+edition = "2021"
+version.workspace = true
```

Comment:
Maybe we should publish this to crates.io? It will be pretty useful for the boltz client crate as well :)